themes/digital.gov/src/js/summary-box.js

```diff
@@ -10,7 +10,7 @@
 
   const guideSummaryList = guideSummary.querySelector(".usa-list");
   const pageHeaders = document.querySelectorAll(
-    "h2:not(.usa-summary-box__heading, .dg-guide__content-header-title)"
+    "h2:not(.usa-summary-box__heading, .dg-guide__content-header-title, .dg-glossary__header)"
```

Comment:
This query acts as "blacklist" of headers to not in the `summary box` component depending on the page it is called from.